src/fabric-1.19.2/livingentity.definition.yaml

```diff
@@ -10,6 +10,8 @@ templates:
     condition: hasSpawnEgg
     deleteWhenConditionFalse: true
 
+unmodifiable_ai_bases: [ Bat, MagmaCube, Slime ]
```

Comment:
I have already fixed this, so you can remove it.

Reply:
done!